sql/codegen.go

```diff
@@ -213,32 +207,15 @@ STEP = 1000
 NUM_BUCKETS=160000
 EMBEDDING_WIDTH=128
 
-{{if eq .Driver "mysql"}}
-db = connect(user="{{.User}}",
-            passwd="{{.Password}}",
-            {{if ne .Database ""}}database="{{.Database}}",{{end}}
-            host="{{.Host}}",
-            port={{.Port}})
-{{else if eq .Driver "sqlite3"}}
-db = connect({{.Database}})
-{{else if eq .Driver "hive"}}
-db = connect(user="{{.User}}",
-            password="{{.Password}}",
-            {{if ne .Database ""}}database="{{.Database}}",{{end}}
-            host="{{.Host}}",
-            port={{.Port}})
+driver="{{.Driver}}"
+{{if ne .Database ""}}
+database="{{.Database}}"
```

Comment:
Can we move this if-else logic into db.py? Something like

```python
def connect(..., database, ...):
    if database == "":
        database = None
    ...
```

Reply:
Ok